client/webpack.server.rails.build.config.js

```diff
@@ -1,90 +1,38 @@
-/* eslint comma-dangle: ["error",
-  {"functions": "never", "arrays": "only-multiline", "objects": "only-multiline"} ],
-  global-require: 0,
-  import/no-dynamic-require: 0,
-  no-console: 0  */
-
-// Common webpack configuration for server bundle
-const { resolve } = require('path');
+const environment = require('./environment');
+const merge = require('webpack-merge');
+const devBuild = process.env.NODE_ENV === 'production' ? 'production' : 'development';
 const webpack = require('webpack');
-const webpackConfigLoader = require('react-on-rails/webpackConfigLoader');
-
-const configPath = resolve('..', 'config');
-const { output } = webpackConfigLoader(configPath);
 
-const devBuild = process.env.NODE_ENV !== 'production';
+// React Server Side Rendering webpacker config
+// Builds a Node compatible file that React on Rails can load, never served to the client.
```

Comment:
why is this file changing?

for just updating the gem, don't change these

do the upgrade steps suggest you need this?